Citizen-Science/src/screens/LoginScreen.tsx

```diff
@@ -103,22 +108,24 @@ const LoginScreen: React.FC = () => {
                     backgroundColor="#5EA1E9"
                     //TODO: potentially redirect to home page after account creation
                     onPress={() => handleLoginPress()}
-                    position={{ horizontal: responsiveWidth(15), vertical: responsiveHeight(75) }}
+                    position={{ top: 12, left: 0 }}
                 />
 
                 {/* Giving Users the option to signup if they are not registered */}
                 <Text style={{fontSize: responsiveFontSize(2), position: 'absolute', top: responsiveHeight(84)}}>
                     Don't have an account?{' '}
                     <TouchableOpacity onPress={() => navigation.navigate('Signup')}>
-                        <Text style={{fontSize: responsiveFontSize(2),  color: '#5EA1E9', textDecorationLine: 'underline' }}>
+                        <Text style={{fontSize: responsiveFontSize(2),  color: '#5EA1E9', top: responsiveHeight(0.4) }}>
```

Comment:
On my phone the text aligns with each other when the value is .65 instead of .4 can you verify?